internal/wasm/module.go

```diff
@@ -662,6 +664,15 @@ type DataSegment struct {
 	Init             []byte
 }
 
+// IsPassive returns true if this data segment is "passive" in the sense that memory offset and
+// index is determined at runtime and used by "memory.init" instruction in the bulk memory
+// operations proposal.
+//
+// See https://github.com/WebAssembly/spec/blob/main/proposals/bulk-memory-operations/Overview.md
```

Comment:
use 2.0 links except when discussing the feature flag or something that is more general than an op like "memory.init" PS use XXXName instead of quoted constants where possible.